Engine/DataFeeds/SubscriptionDataReader.cs

```diff
@@ -202,6 +202,22 @@ object IEnumerator.Current
                 }
             }
 
+            // load up the map and factor files for underlying of equity option
+            if (!config.IsCustomData && config.SecurityType == SecurityType.Option)
```

Comment:
see my comment on price flag property below. I think symbol mapping is an equity specific feature unless we need it for something else.